src/magic_folder/test/test_local_snapshot.py

```diff
@@ -244,7 +244,7 @@ def test_create_snapshots(self, filenames, data):
         files = []
         for filename in filenames :
             file = self.magic.child(filename)
-            content = data.draw(binary())
+            content = snapshot_data.draw(binary())
             file.asBytesMode("utf-8").setContent(content)
```

Comment:
The changes here might be superfluous.  For what it's worth, the parameter that was named `data` before isn't particularly tied to the snapshot data itself.  It is a thing that can draw examples from strategies.  In the `data.draw(binary())` expression, it's `binary()` that is the snapshot data strategy.

If the motivation for the change was the collision between the module-scope `data` name and the `data` parameter, I'm sympathetic.  I'm not sure what a good name improvement is though.  A good non-name-improvement would be to replace the strategy for `filenames` with something like:

```
dictionaries(
    path_segments().map(lambda p: p.encode("utf-8")),
    binary(),
)
```
to produce a dictionary mapping filenames to their contents without using the `data` strategy at all.  The `unique` constraint remains implicitly by virtue of dict keys having to be unique always.

I don't think there is a good reason for the way the test is written now apart from unfamiliarity with different strategy tricks like this.

Reply:
yes, i was just trying to get rid of the name shadowing